invenio_records_marc21/services/services.py

```diff
@@ -85,6 +90,47 @@ def update_draft(
         data = self._create_data(identity, data, metadata, access)
         return super().update_draft(id_, identity, data, revision_id)
 
+    def _lift_embargo_from(self, record):
+        """Lifts embargo from record or draft."""
+        if not record.access.embargo.lift():
+            raise EmbargoNotLiftedError(record["id"])
+        record.access.protection.metadata = "public"
+        record.access.protection.files = "public"
+
+    def _draft_access_field_was_modified(self, draft, record):
+        """Returns True if draft's access field was modified."""
+        return draft.get('access') == record.get('access')
+
+    def lift_embargo(self, _id, identity):
+        """Lifts embargo from the record and updates draft."""
+        # Get the record
+        record = self.record_cls.pid.resolve(_id)
+
+        # Check permissions
+        self.require_permission(identity, "lift_embargo", record=record)
+
+        lifted_embargo_from_draft = False
+        # Check if record has already a draft
+        if record.has_draft:
+            draft = self.draft_cls.pid.resolve(_id, registered_only=False)
+            # If the draft has no modifications in the access field the
+            # embargo is lifted
+            if self._draft_access_field_was_modified(draft, record):
+                # Lifts embargo from draft
+                self._lift_embargo_from(draft)
+                lifted_embargo_from_draft = True
+
+        # Lifts embargo from record
+        self._lift_embargo_from(record)
+        # Commit and index
+        record.commit()
```

Comment:
is there no function that does what is written in that comment? to commit all changes and reindex?